packages/lib/server-only/document/complete-document-with-token.ts

```diff
@@ -69,6 +70,16 @@ export const completeDocumentWithToken = async ({
     },
   });
 
+  const numberOfRecipients = await prisma.recipient.count({
+    where: {
+      documentId: document.id,
+    },
+  });
+
```

Comment:
I think this should filter on those that haven't signed and make sure it's `> 1` otherwise the final recipient will get a double email of pending + completed.